02_Working_With_ASDF_Files/02-Solutions.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>Why is this commented?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I think they all need some explanation. I was planning to uncomment one by one and explain.</p>